go.mod

```diff
@@ -19,5 +19,41 @@ require (
 	go.etcd.io/etcd/client/v3 v3.5.0
 	golang.org/x/net v0.0.0-20210917221730-978cfadd31cf
 	google.golang.org/grpc v1.40.0
+	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
```

Comment:
Is this require needed?

Reply:
it's added by `go mod tidy`. For some reason now after I run `go test ./...` and then `go mod tidy` no more `go: updates to go.mod needed; to update it:`